src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine.kt

```diff
@@ -1067,6 +1078,7 @@ open class TradingStateMachine(
         var configs = state?.configs
         var input = state?.input
         var transferStatuses = state?.transferStatuses?.toIMutableMap()
+        var trackStatuses = state?.trackStatuses?.toIMutableMap()
```

Comment:
why does this need to be an `IMutableMap`? (`IMutableMap` is only needed for types that are consumed directly by JavaScript, internally in abacus we can just use `Map` and `.toMap()`

Reply:
i was under the impression that this state is exposed to the js client, no? this is the `update` fn that constructs the `PerpetualState` object so all the maps are `IMap`s

Reply:
or are we intentionally selective about what abacus state items get exposed to the JS client